exp/operator/main.go

```diff
@@ -23,7 +23,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
-	"k8s.io/klog/klogr"
+	"k8s.io/klog/v2/klogr"
```

Comment:
We should have a verification check that disallows any import path that's non-v1

Reply:
+1